ansible_builder/main.py

```diff
@@ -344,12 +345,6 @@ def create_folder_copy_files(self):
             dest = os.path.join(self.build_context, CONTEXT_BUILD_OUTPUTS_DIR, new_name)
             copy_file(requirement_path, dest)
 
-        # copy introspect.py file from source into build context
-        copy_file(
-            ansible_builder.introspect.__file__,
-            os.path.join(self.build_outputs_dir, 'introspect.py')
-        )
```

Comment:
![what would you say you do here](https://media.giphy.com/media/b7MdMkkFCyCWI/giphy.gif)